d2lbook/config_default.ini

```diff
@@ -136,6 +136,11 @@ latex_logo =
 # Bibtext bibfile
 bibfile =
 
+# Specify the url of external latex resources.
+latex_url = 
+
+# Specify the latex style. We now support "cambridge" and the defaut sphinx style.
+style = 
```

Comment:
Can you set the default value here？So users don't need to specify style = Default if they use default style.

Reply:
they actually do not need to put a default there. it can be empty.

Reply:
Agreed that leaving it empty without any arbitrary default string is better